system_tests/bigtable.py

```diff
@@ -85,18 +85,17 @@ def _operation_finished(result):
 
 
 def _retry_on_unavailable(exc):
-    """Retry only AbortionErrors whose status code is 'UNAVAILABLE'."""
-    from grpc.beta.interfaces import StatusCode
-    return exc.code == StatusCode.UNAVAILABLE
+    """Retry only errors whose status code is 'UNAVAILABLE'."""
+    from grpc import StatusCode
+    return exc.code() == StatusCode.UNAVAILABLE
 
 
 def setUpModule():
-    from grpc.framework.interfaces.face.face import AbortionError
+    from grpc._channel import _Rendezvous
     _helpers.PROJECT = TESTS_PROJECT
     Config.CLIENT = Client(admin=True)
     Config.INSTANCE = Config.CLIENT.instance(INSTANCE_ID, LOCATION_ID)
-    Config.CLIENT.start()
-    retry = RetryErrors(AbortionError, error_predicate=_retry_on_unavailable)
+    retry = RetryErrors(_Rendezvous, error_predicate=_retry_on_unavailable)
```

Comment:
I was thinking we would avoid the "private" class in system tests, at least (maybe we use the `if isinstance(exc, Call)` bit inside the predicate?